packages/office-ui-fabric-react/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx

```diff
@@ -1,298 +1,242 @@
-        key="normal"
-        inputProps={{
-          onBlur: () => console.log('onBlur called'),
-          onFocus: () => console.log('onFocus called'),
-          'aria-label': 'People Picker',
-        }}
-        componentRef={this._picker}
-        headerComponent={this._renderHeader()}
-        focusZoneProps={this._focusZoneProps}
-      />
-    );
-  }
-
-  private _renderHeader(): JSX.Element {
-    return (
-      <div className={this._classNames.to} data-is-focusable={true}>
-        To:
-      </div>
-    );
-  }
-
-  private _getEditingItemText = (item: IExtendedPersonaProps): string => {
+const startsWith = (text: string, filterText: string): boolean => {
+  return text.toLowerCase().indexOf(filterText.toLowerCase()) === 0;
+};
+const classNames = mergeStyleSets({
+  picker: { maxWidth: 400, marginBottom: 15 },
+  headerItem: {
+    borderBottom: '1px solid ' + theme.palette.neutralLight,
+    padding: '8px 12px',
+  },
+  footerItem: {
+    borderBottom: '1px solid ' + theme.palette.neutralLight,
+    height: 60,
+    paddingLeft: 12,
+  },
+  to: { padding: '0 10px' },
+});
+
+const focusZoneProps = {
+  shouldInputLoseFocusOnArrowKey: () => true,
+  handleTabKey: FocusZoneTabbableElements.all,
+};
+
+export const ExtendedPeoplePickerBasicExample: React.FunctionComponent = () => {
+  const picker = React.useRef<ExtendedPeoplePicker>(null);
+  const [peopleList, setPeopleList] = React.useState<IPersonaProps[]>(people);
+  const [mostRecentlyUsed, setMostRecentlyUsed] = React.useState<IPersonaProps[]>(mru);
+  const [searchMoreAvailable, setSearchMoreAvailable] = React.useState<boolean>(true);
+
+  const getEditingItemText = (item: IExtendedPersonaProps): string => {
     return item.text as string;
   };
 
-  private _onSetFocusButtonClicked = (): void => {
-    if (this._picker.current) {
-      this._picker.current.focus();
-    }
-  };
+  const onSetFocusButtonClicked = React.useCallback((): void => {
+    picker.current?.focus();
+  }, []);
 
-  private _onExpandItem = (item: IExtendedPersonaProps): void => {
-    const picker = this._picker.current;
-    const selectedItemsList = picker && picker.selectedItemsList.current;
+  const onExpandItem = (item: IExtendedPersonaProps): void => {
+    const selectedItemsList = picker.current?.selectedItemsList.current;
     if (selectedItemsList) {
-      (selectedItemsList as SelectedPeopleList).replaceItem(item, this._getExpandedGroupItems(item));
+      (selectedItemsList as SelectedPeopleList).replaceItem(item, getExpandedGroupItems(item));
     }
   };
 
-  private _onRemoveSuggestion = (item: IPersonaProps): void => {
-    const { peopleList, mostRecentlyUsed: mruState } = this.state;
+  const onRemoveSuggestion = (item: IPersonaProps): void => {
     const itemIndex = peopleList.indexOf(item);
-    const itemMruIndex = mruState.indexOf(item);
-
+    const itemMruIndex = mostRecentlyUsed.indexOf(item);
     const stateUpdate = {} as IPeoplePickerExampleState;
```

Comment:
Instead of making a stateUpdate object, just do the `setPeopleList` and `setMostRecentlyUsed` calls inline